ui/tests/integration/components/json-editor-test.js

```diff
@@ -29,13 +30,16 @@ module('Integration | Component | json-editor', function (hooks) {
     this.set('onFocusOut', sinon.spy());
     this.set('json_blob', JSON_BLOB);
     this.set('bad_json_blob', BAD_JSON_BLOB);
+    this.set('long_json', JSON.stringify(createLongJson(), null, `\t`));
     this.set('hashi-read-only-theme', 'hashi-read-only auto-height');
     setRunOptions({
       rules: {
         // CodeMirror has a secret textarea without a label that causes this problem
         label: { enabled: false },
         // TODO: investigate and fix Codemirror styling
         'color-contrast': { enabled: false },
+        // failing on .CodeMirror-scroll
+        'scrollable-region-focusable': { enabled: false },
```

Comment:
Good question. This is an ally testing thing—copied the same thing from Chelsea's fixes, here's [one](https://github.com/hashicorp/vault/blob/main/ui/tests/integration/components/database-role-setting-form-test.js#L108) example but there are several more. From what I can tell it's because I'm testing just the component and now it scrolls.